code/_globalvars/lists/mobs.dm

```diff
@@ -24,3 +24,6 @@ GLOBAL_LIST_EMPTY(spidermobs)				//all sentient spider mobs
 
 GLOBAL_LIST_EMPTY(language_datum_instances)
 GLOBAL_LIST_EMPTY(all_languages)
+
+//Global list of monkeys, used for monkeycap.
+GLOBAL_LIST_EMPTY(monkeys)
```

Comment:
think about all the things you use this list for, then take a good long hard look in the mirror and think about your life choices up to this point.

then when you've done all that, turn this into a counter that is incremented and decremented on monkey intialize and Destroy/Death

Reply:
fixed